Project.toml

```
@@ -38,7 +38,7 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 
 [compat]
 AbstractMCMC = "4"
-AdvancedHMC = "0.3.0, 0.4"
+AdvancedHMC = "0.3.0, 0.4, 0.5"
```

Do to the changes below and since the PR requires https://github.com/TuringLang/AdvancedHMC.jl/pull/342, this should be changed to
```suggestion
AdvancedHMC = "0.5.2"
```

---

Doesn't the change in inference/hmc.jl add a function call that does not work on 0.3 and 0.4? Did HMCKernel exist in these old versions?